app/lib/features/space/pages/shell_page.dart

```diff
@@ -113,7 +113,7 @@ class _ShellToolbar extends ConsumerWidget {
                 onTap: () => context.pushNamed(
                   Routes.editSpace.name,
                   pathParameters: {'spaceId': spaceId},
-                  extra: space,
+                  extra: spaceId,
```

Comment:
why have that twice?

Reply:
The ```extra``` object as ```id```` is passed as data on the edit page to fetch the space. ```pathParameters``` is just for routing to correct space.